pages/linux/n.md

```diff
@@ -0,0 +1,23 @@
+# n
+
+> Tool to manage multiple node versions.
+
+- Install a given version of node. If the version is already installed, it will be activated:
+
+`n {{version}}`
+
+- Display installed versions and activate any version:
```

Comment:
No. It displays  list of versions and you can use the arrow keys to select a version and then activate the one you want.

Yea, I agree that part can be clarified.